### PR TITLE
Added Bandisoft.Bandizip v6.29

### PR DESCRIPTION
**Pull request by ceynri**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14301)

### CODE REVIEW

File: manifests/b/Bandisoft/Bandizip/6.29/Bandisoft.Bandizip.installer.yaml
Comment (OfficialEsco) [inline manifests/b/Bandisoft/Bandizip/6.29/Bandisoft.Bandizip.installer.yaml:54]:
```suggestion
    InstallerUrl: https://dl.bandisoft.com/bandizip.old/BANDIZIP6-SETUP.EXE
```

File: manifests/b/Bandisoft/Bandizip/6.29/Bandisoft.Bandizip.installer.yaml
Comment (OfficialEsco) [inline manifests/b/Bandisoft/Bandizip/6.29/Bandisoft.Bandizip.installer.yaml:58]:
```suggestion
      SilentWithProgress: /s
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by ceynri** [comment]
Can anyone help me take a look? I’m not sure what happened. It’s no problem to install it locally 😓.

**Comment by OfficialEsco** [comment]
The error indicates that the system shit itself, nothing wrong with the PR, it will most likely get approved if you repush the package or a maintainer does `/azp run`

**Comment by ItzLevvie** [comment]
If you close the PR and then open it, it should automatically start up the pipelines.

**Comment by OfficialEsco** [comment]
Hey i just realized what was wrong, the silent switch MOST be `/S` not `/s`

**Comment by ceynri** [comment]
> Hey i just realized what was wrong, the silent switch MOST be `/S` not `/s`
> Seems like you just realized that too😄

Yes, I just noticed it and I am preparing to resubmit 🤣

**Pull request merged**
